### ISSUE TITLE
Request a higher contrast theme

### ISSUE THREAD

**Issue by bigshans** [marktext/marktext#3221]
### Describe your feature request
Request a higher contrast theme.

### What problem does this feature solve?

I've tried several themes built into the app but none of them have very high contrast. It's not very comfortable for my eyes. My eyesight isn't very good, and I guess it's not a good thing for other people who don't have very good eyesight.

**Comment by Osndok** [comment]
Duplicates #1640, but yeah... it's a real issue. I'm sure the soft gray text looks beautiful to most.

**Comment by SamuelMiller** [comment]
I agree. I find the poor contrast on my Macbook 2012 hard to work with.  While I like the app, I will have to use another one until a light theme with more contrast (a whiter background and darker text) is added or custom themes are available. My guess is that it wouldn't take a programmer very long to at least add a higher contrast theme.  I would greatly appreciate it.
